sqlmesh/dbt/profile.py

```diff
@@ -26,19 +26,27 @@ def __init__(
         path: Path,
         target_name: str,
         target: TargetConfig,
+        project_model_config: t.Dict[str, t.Any],
```

Comment:
Is it possible to put this in the `DbtContext` instead of adding to the profile, which is a dbt-ism?